### ISSUE TITLE
Browse mode feature request - random and unlabelled

### ISSUE THREAD

**Issue by dobkeratops** [bbernhard/imagemonkey-core#250]
Similar to #203

It would be useful to have a way to search for unlabelled images,

Suggestion 1: some special keywords that work in the search bar: “unlabelled”, “random”,”uploads”(the most recently uploaded images),”history” (the images you have visited recently)

Suggestion 2: if you hit go with an empty request, fill it with one of those, and carry out that search 

Unified mode and add labels could have a different default?

..in unified mode, initialise it to a truly random set (then you only bother entering a specific search term if you didn’t see any interesting annotate life images straight away)

.. In add labels mode , the default could be unlabelled (ie the images in most urgent need of labelling..)

Could unlabelled +random be a default for both? (Ie show unlabelled images first, then if there aren’t any, show random images?.. or a random mix ?

**Comment by bbernhard** [comment]
short update: It's now possible to query the database for unlabeled images. See: 

![Selection_025](https://user-images.githubusercontent.com/5710631/67711485-61b90d80-f9c2-11e9-9deb-c9e2652f3fca.png)

The "order by random" will follow :)

**Comment by dobkeratops** [comment]
That’s awesome !  I’d been killing time doing manual scrapes .. theres 20000 unlabelled images now. there’s quite a bit of variety scattered between a lot of samey examples.. so it’ll be great to go back and browse to pick out the images that stand out

**Comment by bbernhard** [comment]
Yeah, we are already at 70k images - that's really awesome! Many, many thanks for all the contributions and the hard work you put into! It's really great to see the dataset grow!

btw: The free labeling is also almost done (a few minor issues & improvements left) and will go live soon. After that, I'll probably back off a bit from developing new features, as I need to prepare the server move. I'll move the site to a different hosting provider where we get more disk space (we are currently at ~60% disk space usage) and more power. :)
